Dockerfile

```diff
@@ -4,10 +4,23 @@ FROM golang:1.14.2 AS builder
 WORKDIR /go/src/github.com/gardener/logging
 COPY . .
 
-RUN  make plugin
+RUN  make build
+
+############# base
+FROM eu.gcr.io/gardener-project/3rd/alpine:3.12.1 AS base
+
 #############      carrier       #############
-FROM alpine:3.12.0 AS carrier
+FROM base AS carrier
 
 COPY --from=builder /go/src/github.com/gardener/logging/build /source/plugins
 
 WORKDIR /
+
+#############      curator       #############
+FROM base AS curator
+
+COPY --from=builder /go/src/github.com/gardener/logging/build/curator /curator
+
+WORKDIR /
+
+ENTRYPOINT [ "/curator" ]
```

Comment:
I think the pipeline [definition](https://github.com/gardener/logging/blob/2775bfd706e79784fa5db94a79baab28f0029acc/.ci/pipeline_definitions#L11-L14) has to be updated, too. 
See https://github.com/gardener/gardener-extension-provider-aws/blob/10893fca1ef158e3d040cdb09a10cb430dfca7cb/.ci/pipeline_definitions#L10-L19 for example